Gemfile.lock

```diff
@@ -2,14 +2,12 @@ PATH
   remote: .
   specs:
     growthbook (1.0.0)
-      fnv (~> 0.2.0)
```

Comment:
This library has 1 function we're using so I copied the class over, deleted the dead code and am using that. This is because it seems pretty complex to stub type definitions for libraries that do not provide their own type defs, and lack of type defs for all libraries will fail the type checking.